### PR TITLE
fabric network adaptor: support for FSC-based endorsement

### PR DESCRIPTION
**Pull request by adecaro**
This PR does the following: It allows the developer to configure FSC nodes as endorsers of the Token Chaincode.
In this case, the FSC node will not run directly the chaincode but its logic directly.

### CODE REVIEW

File: docs/core-token.md [hunk 1]
```diff
@@ -51,6 +51,25 @@ token:
...
+            # This section is dedicated to the configuration of the endorsement of the token chaincode by
+            # other FSC nodes.
+            endorsement:
+              # Is this node an endorser?: true/false
+              endorser: true
+              # If this node is an endorser, which Fabric identity should be used to sign the endorsement?
+              # If empty, the default identity will be used
+              id: 
+              # A comma-separated list of FSC node identifiers that must be contacted to obtain the endorsement 
+              endorsers: {{ Endorsers }}
```
Comment (alexandrosfilios) [inline docs/core-token.md:71]: Why comma separated and not a list?

File: integration/nwo/token/fabric/opts.go [hunk 2]
```diff
@@ -21,3 +26,35 @@ func IsFabricCA(tms *topology.TMS) bool {
...
+	v, ok := tms.BackendParams["endorsements"]
+	return ok && v.(bool)
+}
+
+func EndorsersToString(tms *topology.TMS) string {
+	v, ok := tms.BackendParams["endorsers"]
+	if !ok {
+		return ""
+	}
+	endorsersStr := strings.Builder{}
```
Comment (alexandrosfilios) [inline integration/nwo/token/fabric/opts.go:46]: We could also use strings.Join I guess, but still i suppose using a list is a cleaner option, unless I'm missing something

File: token/services/network/fabric/endorsement.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+type Endorsement interface {
+	Endorse(context view.Context, tms *token2.ManagementService, requestRaw []byte, signer view.Identity, txID driver.TxID) (driver.Envelope, error)
+}
+
+type FSCEndorsement struct {
+	Endorsers []view.Identity
+}
+
+func (e *FSCEndorsement) Endorse(context view.Context, tms *token2.ManagementService, requestRaw []byte, signer view.Identity, txID driver.TxID) (driver.Envelope, error) {
```
Comment (alexandrosfilios) [inline token/services/network/fabric/endorsement.go:28]: We could make this a service and inject tms and viewmanager

Comment (adecaro) [reply token/services/network/fabric/endorsement.go:28]: This has the same signature of the method in the Network interface. We need tms at the time of the invocation. The viewManager con be injected, sure.

File: integration/token/common/topology.go [hunk 4]
```diff
@@ -32,6 +32,7 @@ type Opts struct {
 	SDKs            []api.SDK
 	WebEnabled      bool
 	Monitoring      bool
+	WithEndorsers   bool
```
[the remark this answers is not in the document]
Comment (adecaro) [reply integration/token/common/topology.go:35]: renamed

File: integration/nwo/token/fabric/opts.go (hunk 2 above)
Comment (adecaro) [reply integration/nwo/token/fabric/opts.go:46]: okay

File: docs/core-token.md (hunk 1 above)
Comment (adecaro) [reply docs/core-token.md]: changed 👍

**Review by alexandrosfilios — APPROVED** [review]

**Pull request merged**